src/components/Button/index.js

```diff
@@ -0,0 +1,69 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { cx } from "emotion"
+import {
+  wrapper,
+  primaryBtn,
+  secondaryBtn,
+  dangerBtn,
+  ghostBtn
```

Comment:
are secondaryBtn, dangerBtn and ghostBtn used anywhere. If you don't see these lines underlined you should setup eslint in VScode. The rule is that if the variable is not used then it shows up as an error